### PR TITLE
[EDU-3863] uninstall edge node pt

### PR DESCRIPTION
**Pull request by gabriel-azion**
Related issue: [EDU-3863](https://aziontech.atlassian.net/browse/EDU-3863)

[EDU-3863]: https://aziontech.atlassian.net/browse/EDU-3863?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/deploy-jornada/desinstalar-node/desinstalar-node.mdx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+:::caution 
+Uma vez que um edge node é deletado, ele não pode ser recuperado.
+::: 
+
+### Via RTM 
+
+1. [Acesse o RTM](/pt-br/documentacao/produtos/guias/acesso-rtm/) > **Edge Nodes**.
+2. Encontre o edge node que você deseja remover.
+3. Passe o mouse sobre o edge node e clique no ícone de uma lixeira.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/deploy-jornada/desinstalar-node/desinstalar-node.mdx:48]:
```suggestion
3. Passe o cursor sobre o edge node e clique no ícone de uma lixeira.
```

File: src/content/docs/pt-br/pages/deploy-jornada/desinstalar-node/desinstalar-node.mdx [hunk 2]
```diff
@@ -0,0 +1,79 @@
+---
+title: Como desinstalar o Edge Orchestrator Agent
+description: Desinstale o Edge Orchestrator de um edge node e remova-o da listagem de edge nodes disponíveis.
+meta_tags: deploy, orchestrator
+namespace: docs_deploy_uninstall_agent
+menu_namespace: deployMenu
+og_image: ''
+permalink: /documentacao/produtos/guias/deploy/avancado/desinstalar-agent/
+---
+
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/deploy-jornada/desinstalar-node/desinstalar-node.mdx:10]: não tem intro aqui?

File: src/content/docs/pt-br/pages/deploy-jornada/desinstalar-node/desinstalar-node.mdx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+2. Execute a seguinte requisição `DELETE` no seu terminal, substituindo `[TOKEN VALUE]` pelo seu [personal token](/pt-br/documentacao/produtos/guias/personal-tokens/) e `:id` com o ID do edge node:
+
+```bash 
+curl --location --request DELETE 'https://api.azionapi.net/edge_nodes/:id' \
+--header 'Accept: application/json; version=3' \
+--header 'Authorization: Token [TOKEN VALUE]'
+```
+
+:
+:::tip
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/deploy-jornada/desinstalar-node/desinstalar-node.mdx:71]:
```suggestion
:::tip
```

File: src/content/docs/pt-br/pages/deploy-jornada/desinstalar-node/desinstalar-node.mdx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+Saída: 
+
+```bash 
+Edge Orchestrator was uninstalled successfully
+```
+
+A partir deste momento, seu edge node não receberá mais atualizações.
+
+:::note 
+Desinstalar o agente do seu dispositivo não remove o edge node da lista de nodes disponíveis. Para completar o processo, você precisa excluir seu edge node da lista de edge nodes disponíveis.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/deploy-jornada/desinstalar-node/desinstalar-node.mdx:32]:
```suggestion
Desinstalar o agente do seu dispositivo não remove o edge node da lista de nodes disponíveis. Para completar o processo, você precisa [excluir seu edge node]() da lista de edge nodes disponíveis.
```

Comment (MarianaAguilera) [reply src/content/docs/pt-br/pages/deploy-jornada/desinstalar-node/desinstalar-node.mdx:32]: revisando, poderia ser legal adicionar o link desse guia aqui

File: src/content/docs/pt-br/pages/deploy-jornada/desinstalar-node/desinstalar-node.mdx [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+::: 
+
+
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/deploy-jornada/desinstalar-node/desinstalar-node.mdx:35]:
```suggestion
```

File: src/content/docs/pt-br/pages/deploy-jornada/desinstalar-node/desinstalar-node.mdx [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+---
+
+## Exclua um edge node
+
+:::caution
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/deploy-jornada/desinstalar-node/desinstalar-node.mdx:40]:
```suggestion
:::caution [Atenção]
```

File: src/content/docs/pt-br/pages/deploy-jornada/desinstalar-node/desinstalar-node.mdx [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+## Exclua um edge node
+
+:::caution 
+Uma vez que um edge node é deletado, ele não pode ser recuperado.
+::: 
+
+### Via RTM 
+
+1. [Acesse o RTM](/pt-br/documentacao/produtos/guias/acesso-rtm/) > **Edge Nodes**.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/deploy-jornada/desinstalar-node/desinstalar-node.mdx:46]:
```suggestion
1. [Acesse o RTM](/pt-br/documentacao/produtos/guias/como-acessar-o-rtm/) > **Edge Nodes**.
```

File: src/content/docs/pt-br/pages/deploy-jornada/desinstalar-node/desinstalar-node.mdx [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+2. Encontre o edge node que você deseja remover.
+3. Passe o cursor sobre o edge node e clique no ícone de uma lixeira.
+4. Confirme a exclusão digitando a palavra `delete`.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/deploy-jornada/desinstalar-node/desinstalar-node.mdx:49]:
```suggestion
4. Confirme a exclusão digitando a palavra *delete*.
```

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**